utils/common/src/main/kotlin/ProcessCapture.kt

```diff
@@ -60,6 +65,17 @@ class ProcessCapture(
                 message
             }
         }
+
+        /**
+         * Filter out the variables that are not white-listed to be propagated from [environment].
```

Comment:
Nit: We should probably try to avoid the white / black list terms, and rather speak of allow / deny lists.